src/App.js

```diff
@@ -54,6 +57,181 @@ const App = () => {
           </div>
         </div>
       </section>
+      <section className="digits">
+        <h3 className="digits__title">Цифры и факты</h3>
+        <h2 className="digits__subtitle">Про учебу и мозг</h2>
+        <table className="digits__table">
+          <tr>
+            <td>
+              <p className="table__title">86 миллиардов</p>
+              <p className="table__text">Число нейронов в мозге человека</p>
+            </td>
+            <td>
+              <p className="table__title">2.1 миллиарда</p>
+              <p className="table__text">
+                Число нуждающихся в повышении квалификации
+              </p>
+              <p className="bank">Всемирный Банк, 2017</p>
+            </td>
+            <td>
+              <p className="table__title">1000 терабайт</p>
+              <p className="table__text">Объём памяти человека</p>
+            </td>
+            <td>
+              <p className="table__title">500 триллионов</p>
+              <p className="table__text">
+                Число ответственных за обучение нервных синапсов у взрослого
+                человека
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <p className="table__title">420 миллионов</p>
+              <p className="table__text">
+                Число взрослых людей моложе 25 лет, не имеющих образования для
+                трудоустройства
+              </p>
+              <p className="bank">Всемирный Банк, 2017</p>
+            </td>
+            <td>
+              <p className="table__title">17–20 лет</p>
+              <p className="table__text">Пик обучаемости</p>
+            </td>
+            <td>
+              <p className="table__title">1885 год</p>
+              <p className="table__text">Открытие кривой забывания</p>
+            </td>
+            <td>
+              <p className="table__title">1889 год</p>
+              <p className="table__text">Открытие условного рефлекса</p>
+            </td>
+          </tr>
+        </table>
+      </section>
+      <section className="feynman" height="890px">
+        <h1 className="feynman__title">Метод Фейнмана</h1>
+        <h2 className="feynman__subtitle">Выучить и не забыть.</h2>
+        <img
+          className="feynman__portrait"
+          src={feynman}
+          width="867px"
+          height="637px"
+          alt="Portrait of Feynman"
+        />
+        <a href="https://en.wikipedia.org/wiki/Wiki" className="feynman__link">
```

Comment:
Nice to know: if you add `target="_blank"` then browser will open this link in a new window

No need to change something here 👌